src/routes/community/community.controller.ts

```diff
@@ -44,6 +45,22 @@ export class CommunityController {
     return this.communityService.getCampaignById(resourceId);
   }
 
+  @Get('/campaigns/:resourceId/points/:safeAddress')
```

Comment:
After discussing this with Core, this is quite restrictive. The entity returned is actually a flattened `Period` and `Activity` so we should rethink this.

They are opting for `/api/v1/campaigns/{resourceId}/addresses/{address}/periods`, for example.